Add links to VRF v2 youtube video and blog. Fix minor typo and usage … (#636)
smartcontractkit/documentation: docs/Using Randomness/example-contracts.md

## Subscription manager contract

> In this example, the contract operates as a subscription owner and can run functions to add consumer contracts to the subscription. Those contracts need to include the `requestRandomWords()` with the correct coordinator parameters and the correct subscription ID to obtain their own random values and use the subscription balance and the `fulfillRandomWords()` functions.

Reviewer: ```suggestion
In this example, the contract operates as a subscription owner and can run functions to add consumer contracts to the subscription. The consumer contracts must include the `requestRandomWords()` function with the correct coordinator parameters and the correct subscription ID to request random values and use the subscription balance. The consumer contracts must also include the `fulfillRandomWords()` functions to receive the random values.
```
